### ISSUE TITLE
Casing of scene names 

### ISSUE THREAD

**Issue by Patrick1610** [nilsreiter/home-assistant-scenes#2]
Hi! When I use the code you provided in a service call, it only works with a Scene without a space (i.e. Nebula). 
When I try Savanna Sunset or Emerald Isle I get the following error. Also when I use single or double quotes. 
![image](https://github.com/nilsreiter/home-assistant-scenes/assets/6585375/986351af-2d54-4edb-8bc2-daffad913ecb)

What could I be doing wrong?
Regards Patrick

**Comment by Patrick1610** [comment]
I was reading through your code and saw that the second words are no longer written with capitals, so that's what was causing the issue! Works like a charm now, very sorry for the opened issue.

I do have another question though, is it okay to as this here? 
- you have all the names and color codes in the repo, and I found al the images in the Hypfer repo, but they don't have the names of the scenes. Is there a place where I can see what scene is based on what image? I want to use the images in my dashboard, so I need the right ones 😉
- in the last example of a dashboard version, you refer to the image as image.savanna-sunset.jpg. Where do in have to store them to make this work? 

Thank you very much for your time, and have a great day!

**Comment by nilsreiter** [comment]
Casing: At some point I noticed that they are not written in title case in the Hue app, so I changed everything. The second word (if present) is now constantly written in lower case.

The images are stupidly annoying. Many of them are listed in the Hue app, and I just spent half a day following the links. Some images from the Hue app I couldn't find, I think some photographers have taken them down.

My photo setup is:

- All images are named according to the scene name and in 1024x768 format. Some I had to crop. They are stored in `config/www/hue/`, e.g., `config/www/hue/Savanna Sunset.jpg`.
- I started with the image entities, but this is rather cumbersome. I changed the setup to the following: There is a [`folder`](https://www.home-assistant.io/integrations/folder/) sensor that lists all the image paths as one of its attributes. If I know the scene name, the picture path is `/local/hue/SCENENAME.jpg` (and so far, I didn't have any issues with spaces ...)

I should write this up somewhere ...
